### DISCUSSION TITLE
Strokes above or below

### DISCUSSION THREAD

**Discussion by 4l3m4oO** [friction2d/friction#56] [Ideas]
I think it would be useful if we can arrange the stroke of the elements, since we can add one (or more) stroke below, we have to duplicate the object and put it below only with strokes. If it would have an option that can add an stroke (or more)  and arrange it, it will much easier and organized.

![image](https://github.com/friction2d/friction/assets/357488/85c51851-cdaa-4e1e-955b-39abb6c569db)

**Comment by rodlie** [comment]
Stroke alignment is not supported. 

The SVG spec does not support it, skia (the engine in Friction) does not support it and no browser engine support it.

* https://drafts.fxtf.org/fill-stroke-3/#propdef-stroke-align
* https://svgwg.org/specs/strokes/#SpecifyingStrokeAlignment
> not supported in any modern browser, nor are there any known plans to support it.
